server/test/Makefile

```diff
@@ -1,19 +1,30 @@
 CC = gcc
-CFLAGS = -std=c99 -Wall -pedantic -g
```

Comment:
maybe removed in panic ???? Yep, we should have leaved as it is as below
"+CFLAGS = -std=c99 -Wall -pedantic -g -pthread"